src/pose_estimation/scoring/angle_score.py

```diff
@@ -0,0 +1,21 @@
+from score import Score
+from keypoint_statistics import KeypointStatistics
+import numpy as np
+
+
+class AngleScore(Score):
+    # overriding abstract method 
+    def compute_score(first_keypoints: KeypointStatistics, second_keypoints: KeypointStatistics, weights: np.ndarray = None) -> float:
+        """
+        Returns the average angle difference between
+        the two keypoints
+        :param first_keypoints: first set of keypoints
+        :param second_keypoints: second set of keypoints
+        :param weights: weights to apply to each angle
+        :return: angle score
+        """
+        angle_diff = np.abs(first_keypoints.to_list() - second_keypoints.to_list())
```

Comment:
does `to_list` return a numpy array? Maybe you could add a `to_numpy` method in the `Keypoints` class for this to work?

Reply:
Yea it returns a numpy array.